DNN Platform/Components/ClientDependency/Source/DependencySorter.cs

```diff
@@ -10,6 +10,30 @@ namespace ClientDependency.Core
     /// </summary>
     internal static class DependencySorter
     {
+
+        public static IList<IClientDependencyFile> FilterDependencies(IList<IClientDependencyFile> dependencies)
+        {
+            if (dependencies.Any(f => f.Name != ""))
+            {
+                var newList = dependencies.Where(f => f.Name == "").ToList();
+                var frameworks = dependencies.Where(f => f.Name != "").GroupBy(f => f.Name.ToLower());
+                foreach (var framework in frameworks)
+                {
+                    var topPriority = framework.FirstOrDefault(d => d.ForceVersion);
```

Comment:
I'm not sure I follow. But I'm not very particular about this point. I do feel strongly about the use case that a skin dev should have the possibility to say: "ok, we're using this version (and file) of this framework". For the simple reason that the skin designer is the boss in the UI.